Fail with Go unreleased version

Thanks to https://github.com/gopasspw/gopass/blob/master/helpers/postrel/main.go#L274 the tests fail if executed with an RC version of golang, which should not happen:

```
--- FAIL: TestGoVersion (0.00s)
panic: semver: Parse(1.25rc2 X:nodwarf5): No Major.Minor.Patch elements found [recovered, repanicked]

goroutine 4 [running]:
testing.tRunner.func1.2({0x56445bbb36c0, 0xc00002c340})
	/usr/lib/golang/src/testing/testing.go:1872 +0x237
testing.tRunner.func1()
	/usr/lib/golang/src/testing/testing.go:1875 +0x35b
panic({0x56445bbb36c0?, 0xc00002c340?})
	/usr/lib/golang/src/runtime/panic.go:783 +0x132
github.com/gopasspw/gopass/vendor/github.com/blang/semver/v4.MustParse({0x56445ba96da2, 0x12})
	/builddir/build/BUILD/gopass-1.15.16-build/gopass-1.15.16/_build/src/github.com/gopasspw/gopass/vendor/github.com/blang/semver/v4/semver.go:363 +0x17d
github.com/gopasspw/gopass/helpers/postrel.goVersion()
	/builddir/build/BUILD/gopass-1.15.16-build/gopass-1.15.16/_build/src/github.com/gopasspw/gopass/helpers/postrel/main.go:274 +0x55
github.com/gopasspw/gopass/helpers/postrel.TestGoVersion(0xc000003880)
	/builddir/build/BUILD/gopass-1.15.16-build/gopass-1.15.16/_build/src/github.com/gopasspw/gopass/helpers/postrel/main_test.go:68 +0x18
testing.tRunner(0xc000003880, 0x56445bc50f48)
	/usr/lib/golang/src/testing/testing.go:1934 +0xea
created by testing.(*T).Run in goroutine 1
	/usr/lib/golang/src/testing/testing.go:1997 +0x465
exit status 2
FAIL	github.com/gopasspw/gopass/helpers/postrel	0.057s
error: Bad exit status from /var/tmp/rpm-tmp.F37eQW (%check)
    Bad exit status from /var/tmp/rpm-tmp.F37eQW (%check)
```

--- a/helpers/postrel/main.go
+++ b/helpers/postrel/main.go
@@ -270,8 +270,19 @@ func versionFile() (semver.Version, error) {
 	return semver.Parse(strings.TrimSpace(string(buf)))
 }
 
-func goVersion() string {
-	sv := semver.MustParse(strings.TrimPrefix(runtime.Version(), "go"))
+func goVersion(v string) string {
+	v = strings.TrimPrefix(v, "go")
+	sv, err := semver.ParseTolerant(v)
+	if err != nil {
+		// if we can't parse the version, we assume it's a dev version
+		// and return the current major.minor version
+		if len(v) > 3 {
+			sv, err = semver.ParseTolerant(v[:3])
+			if err != nil {
+				panic(err)
+			}
+		}
+	}
 
 	return fmt.Sprintf("%d.%d", sv.Major, sv.Minor)
 }
@@ -286,7 +297,7 @@ func newIntegrationsUpdater(client *github.Client, v semver.Version) (*inUpdater
 	return &inUpdater{
 		github: client,
 		v:      v,
-		goVer:  goVersion(),
+		goVer:  goVersion(runtime.Version()),
 	}, nil
 }
 

--- a/helpers/postrel/main_test.go
+++ b/helpers/postrel/main_test.go
@@ -1,5 +1,5 @@
-//go:build linux
-// +build linux
+//go:build linux || darwin
+// +build linux darwin
 
 package main
 
@@ -65,8 +65,10 @@ func TestVersionFile(t *testing.T) {
 
 // Test goVersion function
 func TestGoVersion(t *testing.T) {
-	version := goVersion()
-	assert.NotEmpty(t, version)
+	for _, v := range []string{"1.15", "1.16", "1.17", "1.25rc2"} {
+		t.Logf("Testing goVersion with %s", v)
+		assert.NotEmpty(t, goVersion(v), v)
+	}
 }
 
 // Test updateWorkflows function

